### PR TITLE
Add homebrew install instructions for nvm compat

### PR DESCRIPTION
**Pull request by ntwb**
Fixes https://github.com/yarnpkg/yarn/issues/3255

### CODE REVIEW

File: lang/en/docs/_installations/mac.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ This will also install Node.js if it is not already installed.
 brew install yarn
 ```
 
+If you use [nvm](https://github.com/creationix/nvm) you should exclude installing NodeJS so that the nvm binaries are used.
```
Comment (Daniel15) [inline lang/en/docs/_installations/mac.md]: Maybe change "nvm binaries" to "nvm's version of Node.js"?

Also, small nit: Use "Node.js", not "NodeJS".

Comment (ntwb) [inline lang/en/docs/_installations/mac.md]: Ha, and done, I've been writing it that way for years, my bad, sorry Node.js

**Comment by Daniel15** [comment]
Thanks!

**Comment by nandorojo** [comment]
Is this no longer published on the docs? Looks like the current installation explanation is a little less clear https://classic.yarnpkg.com/en/docs/install/#mac-stable

Thanks!

**Comment by Daniel15** [comment]
@nandorojo  Homebrew removed the ability to install Yarn without installing Node.js a while back. In fact they removed all customization options for formulae.

**Comment by nandorojo** [comment]
You’re right. I went with the CURL install instead — this may be the best option for nvm users to avoid node installs. Maybe I could add a PR to suggest that?

**Comment by Daniel15** [comment]
Sure, feel free to submit a PR.

**Pull request merged**